.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
   check-workflow:
     name: Check reusable workflow
-    runs-on: ubuntu-22.04
+    runs-on: ubicloud-standard-2-arm
```

Comment:
do the other steps not work on an arm runner, or is it just coincidence that these are arm?

Reply:
that's a good question. I put arm in workflows where I am 99% sure it will work. What I noticed that workflows with building docker images don't work with arm. So we can experiment here